charts/opensearch/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Deprecated
 ### Removed
 ### Fixed
+- Remove whitespace from `admin_dn` in base `values.yaml`.
```

Comment:
Wrong edit, this should be in a new block and I recommand you use 1.3.2 since #128 is using 1.3.1.

Thanks.

Reply:
Probably need to change to 1.4.1 later as #123 is having a bump to 1.4.0 for breaking changes.

Reply:
https://github.com/opensearch-project/helm-charts/pull/130/commits/c1663155b754fdb3e82e507b1c2874db88dbfdf4